lib/eig_trlm.cpp

```diff
@@ -166,7 +175,7 @@ namespace quda
               n_conv, restart_iter, iter);
 
       // Dump all Ritz values and residua if using Chebyshev
```

Comment:
Show we adjust this comment to reflect the change?

Reply:
https://github.com/lattice/quda/pull/1421/commits/431c4ecf2884e7630d8b8b64655a1cf7d0bbe3f7